scripts/Update-QDKVersion.ps1

```diff
@@ -41,48 +41,38 @@ $csFiles | ForEach-Object {
     } | Set-Content -Encoding UTF8 $_.Path
 }
 
+function findAndReplace() {
+    param (
+        [string]$stringToReplace,
```

Comment:
I'd call this variable `$patternToSearch`, since we're not replacing the whole string, just parts of it

Reply:
Thanks for the feedback! I've added your suggestions and started a new pipeline run 🙂